### DISCUSSION TITLE
Turn off deep convection?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#912] [Q&A] [answered]
We are running a test of the model at C3072 and would like to turn off the deep convection scheme. Our first attempt was:
do_deep = .false.
imfdeepcnv = -1 (was =2)
ras = .false.

However, that left the results entirely unchanged. What do we need to do to turn off deep convection?

**Comment by pjpegion** [comment]
I suspect you will need to modify the suite definition file you compile with:
try removing these lines...

      <scheme>GFS_DCNV_generic_pre</scheme>
      <scheme>samfdeepcnv</scheme>
      <scheme>GFS_DCNV_generic_post</scheme>

**Reply by benjamin-cash** [reply]
Thanks @pjpegion. Are those changes in addition to the namelist changes?

**Reply by pjpegion** [reply]
To be safe, yes. I don't understand the interplay between the suite definition files and the namelist options.

**Reply by ShawnCebulaNOAA** [reply]
@benjamin-cash Did this fix the issue?  Please let me know!

Shawn Cebula - EPIC End User Support

**Reply by benjamin-cash** [reply]
Hi @ShawnCebulaNOAA - We ended up abandoning this approach for other reasons so it ended up not being relevant.

**Comment by junwang-noaa** [comment]
The suite_FV3_GFS_v15_thompson_mynn_lam3km.xml used by regional high
resolution runs has the connection turned off, maybe you can use it as a
reference. You do need the namelist changes too.

**Comment by benjamin-cash** [comment, marked as answer]
Doing cleanup on my old questions - going to mark this as answered. Although we never did figure out how to do it we've moved on to other efforts.
